### PR TITLE
Added german

### PR DESCRIPTION
**Pull request by mclucario**
A complete german translation in translation file format v1

### CODE REVIEW

File: src/Language/german.php [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+ * When you see a string is specifically formatted, for example:
+ *  'Type  `@e621searchbot <tags>` , examples:'
+ * do not change it's format (you can see there are two spaces after 'Type' and one space before a colon, this is for better viewing in the Telegram, DO NOT MODIFY IT)
+ *
+ */
+
+$translation = [
+    // Translation specific stuff
+    'language' => 'Deutsch',
+    'language_emoji' => '🇩🇪 🇩🇪',
```
Comment (jacklul) [inline src/Language/german.php]: No need for two emojis.

File: src/Language/german.php [hunk 2]
```diff
@@ -0,0 +1,322 @@
+﻿<?php
+
+/**
+ * @e621searchbot translation template (version 1)
```
Comment (jacklul) [inline src/Language/german.php]: This whole 'helper' comment block can be safely removed from finished translation.

Comment (mclucario) [inline src/Language/german.php]: Alright.

File: src/Language/german.php (hunk 1 above)
Comment (mclucario) [inline src/Language/german.php]: I added them because there are different versions of german like switzer german and stuff like that, y'know.

**Pull request merged**